src/mlpack/methods/perceptron/perceptron_main.cpp

```diff
@@ -297,6 +312,8 @@ static void mlpackMain()
     // Save the predicted labels.
     if (CLI::HasParam("output"))
       CLI::GetParam<arma::Row<size_t>>("output") = std::move(results);
+    if (CLI::HasParam("predictions"))
+      CLI::GetParam<arma::Row<size_t>>("predictions") = std::move(results);
```

Comment:
We can't `std::move()` twice (just like the other file).

Reply:
Changed